src/App.js

```diff
@@ -5,16 +5,48 @@ import Button from "@mui/material/Button";
 import CssBaseline from "@mui/material/CssBaseline";
 import Box from "@mui/material/Box";
 import Container from "@mui/material/Container";
-import TodoBox from "./components/TodoBox";
+import TodoForm from "./components/TodoForm";
+import Typography from "@mui/material/Typography";
+import Paper from "@mui/material/Paper";
+import AppBar from "@mui/material/AppBar";
+import Toolbar from "@mui/material/Toolbar";
+import { createTheme, ThemeProvider } from "@mui/material/styles";
+// import CheckboxList from "./components/CheckboxList";
 
 function App() {
+  const theme = createTheme();
+
   return (
-    <div>
-      <CssBaseline />
+    <ThemeProvider theme={theme}>
       <Container>
-        <TodoBox />
+        <CssBaseline />
+
+        <AppBar
+          position="absolute"
+          color="default"
+          elevation={0}
+          sx={{
+            position: "absolute",
+            borderBottom: (t) => `1px solid ${t.palette.divider}`,
+          }}>
+          <Toolbar>
+            <Typography variant="h4" color="inherit" noWrap>
+              To Do App
+            </Typography>
+          </Toolbar>
+        </AppBar>
+        <Container component="main" maxWidth="sm" sx={{ mb: 16 }}></Container>
+        <Paper
+          variant="outlined"
+          sx={{ my: { xs: 3, md: 16 }, p: { xs: 2, md: 5 } }}>
+          <Typography component="h1" variant="h4" align="center">
+            To do App
+          </Typography>
+          <TodoForm />
+          {/* <CheckboxList /> */}
```

Comment:
This kind of comments into the JSX is also considered a bad practice.

Reply:
done